component/async/component_test.go

```diff
@@ -105,7 +129,7 @@ func TestRun_ReturnsError(t *testing.T) {
 	err := run(context.Background(), t, &builder)
 
 	assert.Error(t, err)
-	assert.True(t, strings.Contains(err.Error(), errConsumer.Error()))
+	assert.True(t, errors.Is(err, errConsumer))
```

Comment:
`assert.True(t, errors.Is(err, errConsumer))` also covers the fact that the error isn't `nil`. Same thing for other tests that have both `assert.Error` and `assert.True(errors.Is)`

Reply:
right, will remove the asser.Error in next commit